python/ray/serve/controller.py

```diff
@@ -715,6 +745,25 @@ def get_all_deployment_statuses(self) -> List[bytes]:
         statuses = self.deployment_state_manager.get_deployment_statuses()
         return [status.to_proto().SerializeToString() for status in statuses]
 
+    def get_user_deployed_app_configs(self) -> Dict[str, ServeApplicationSchema]:
```

Comment:
nit: `get_app_configs(self)`?

Reply:
Sorry, this was an old function that I'm not using anymore. I am using the existing `get_app_config` directly - will remove this function.